2024Robot_Team3484/src/main/include/subsystems/LauncherSubsystem.h

```diff
@@ -0,0 +1,44 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+#ifndef LAUNCHERH
+#define LAUNCHERH
+
+#include <Constants.h>
+
+#include <frc2/command/SubsystemBase.h>
+
+#include <rev/CANSparkMax.h>
+
+#include <units/angle.h>
+#include <units/angular_velocity.h>
+
+class LauncherSubsystem : public frc2::SubsystemBase {
+    public:
+        LauncherSubsystem(
+            int MOTOR_LEFT_CAN_ID, 
+            int MOTOR_RIGHT_CAN_ID,
```

Comment:
Confirm naming conventions here. I know all caps is used to designate constants. For function/constructor parameters I would use PascalCase.